app/views/profiles/_preview.html.erb

```diff
@@ -0,0 +1,30 @@
+<turbo-frame id=<%= profile.id %>>
+  <div class="card">
+    <div class="card-header">
+    </div>
+    <div class="card-content">
+      <div class="two-column">
+        <div>
+          <%= image_tag profile.user.avatar_url, class:"profile-avatar" %>
```

Comment:
We might have to wrap this avatar for safety – what if there isn't one present? The Github API call may have failed, or in the future we might want to give the user the option to remove their avatar. 

A default placeholder type image may be a good idea?